cln-rpc/src/model.rs

```diff
@@ -2082,6 +2082,22 @@ pub mod responses {
 	    pub status: PayStatus,
 	}
 
+	#[derive(Clone, Debug, Deserialize, Serialize)]
+	pub struct ListnodesNodesOption_will_fund {
```

Comment:
```suggestion
	pub struct ListnodesNodesOption_will_fund {
```

Yeah I think it is a bug